admin/views/redirects.php

```diff
@@ -0,0 +1,237 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ * @since   19.0
+ */
+
+if ( ! defined( 'WPSEO_VERSION' ) ) {
+	header( 'Status: 403 Forbidden' );
+	header( 'HTTP/1.1 403 Forbidden' );
+	exit();
+}
+
+?>
+<div class="wrap yoast wpseo-admin-page page-wpseo_redirects">
+	<h1 id="wpseo-title"><?php echo \esc_html( \get_admin_page_title() ); ?></h1>
+	<div class="wpseo_content_wrapper" style="position: relative;">
+		<div style="position: absolute;top: 0;bottom: 0;left: 0;right: 0;z-index: 100; display: flex;justify-content: center;align-items: center;background: radial-gradient(#ffffffcf 20%, #ffffff00 50%);">
+			<a class="yoast-button-upsell" href="<?php echo \esc_url( WPSEO_Shortlinker::get( 'https://yoa.st/4e0/' ) ); ?>" target="_blank">
+				<?php
+				\esc_html_e( 'Unlock with Premium', 'wordpress-seo' )
+					// phpcs:ignore WordPress.Security.EscapeOutput -- Already escapes correctly.
+					. WPSEO_Admin_Utils::get_new_tab_message();
```

Comment:
I would do that as:
```
				\esc_html_e( 'Unlock with Premium', 'wordpress-seo' );
					// phpcs:ignore WordPress.Security.EscapeOutput -- Already escapes correctly.
				echo WPSEO_Admin_Utils::get_new_tab_message();
```
because `WPSEO_Admin_Utils::get_new_tab_message()` doesn't echo anythign on itself.

Reply:
Yes, thanks, my fault: I was changing all the other occurrences and applied the change here where it wasn't needed at all